helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -788,14 +788,18 @@ protected void setupInstances(String clusterName, int[] instances) {
     }
   }
 
-  protected void runPipeline(ClusterEvent event, Pipeline pipeline) {
+  protected void runPipeline(ClusterEvent event, Pipeline pipeline, boolean shouldThrowException)
```

Comment:
How about giving us a break by keeping the existing method with shouldThrowException default to be false?
So you have much less changed files.